docs/user-interface/skills_knowledge_contributions.md

```diff
@@ -0,0 +1,83 @@
+---
+title: InstructLab UI Skills & Knowledge contributions
+description: Steps to contribute to the community model via UI.
+logo: images/ilab_dog.png
+---
+
+The UI Simplifies the process for Skills & Knowledge contributions by:
+
+* Minimising risk of human error when writing YAML by using the web form. 
+
+* Directly submit a github pull request with a press of a button.
+
+When the form is filled out, you also are given the option to download the YAML and attribution files to your local machine, and to view the form in its original YAML structure before submission.
+
+You can view all your submissions on the dashboard page.
+
+!!! warning
+    Even when running the UI locally, you must be logged in via github to successfully submit your Knowledge and Skills contributions. You can still fill out the form, and download the YAML and attribution files.
+
+For tips on writing Skills & Knowledge contributions, please visit the documentation under the [Taxonomy](/taxonomy/) heading.
+
+## Skill Contributions
+
+### Author Information
+
+Enter your github email address, and your full name, which will be tied to your contribution.
+
+### Skill Information
+
+Give a brief description on what skill you would like to teach.
+
+### Taxonomy Directory Path
+
+Specify where in the taxonomy this skill would belong in. As the taxonomy fills, it will become more complex, make sure to give a concise location to keep the taxonomy organised.
+
+### Seed Examples
+
+For a skill contribution, the seed example only requires a question and answer, but depending on what skill you wish to add, adding context for a skill may be necessary, For tips on writing Skills contributions, please visit the documentation under the [Skills Guide](/taxonomy/skills/skills_guide) heading.
+
+### Attribution
+
+Linking the source you used for information. Wikipedia articles change overtime, make sure to add the oid from the wikipedia article, which you can find by clicking on `View history`, and select the relevant version.
+
+## Knowledge Contributions
```

Comment:
I am just wondering if we put skill and knowledge contributions sections in it's own separate files ? That way we can manage both of them separate and doc won't get too long. wdyt?

Reply:
I put them together since separate pages would have been short, but I agree that it would be easier to manage if they were separate; I'll make the changes